apps/daffio/src/app/content/docs/api-docs/models/api-doc-reference.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * An object for a reference to an API document.
+ */
+export interface DaffioApiDocReference {
+  name: string;
+  title: string;
+  path: string;
+  docType: string;
```

Comment:
Can this be more strongly typed?

Reply:
Like an enum or what?

Reply:
A simple `export type DaffApiDocType = "Component" | "Class"... ` would suffice.